cpu.go

```diff
@@ -1,17 +1,29 @@
-package internal
+package ebpf
 
 import (
 	"fmt"
 	"os"
 	"strings"
+
+	"github.com/cilium/ebpf/internal"
 )
 
 // PossibleCPUs returns the max number of CPUs a system may possibly have
 // Logical CPU numbers must be of the form 0-n
-var PossibleCPUs = Memoize(func() (int, error) {
+var PossibleCPUs = internal.Memoize(func() (int, error) {
```

Comment:
Can you make the variable unexported instead, and wrap it in a real function? Doing it this way makes the documentation really wonky. Example: https://pkg.go.dev/github.com/cilium/ebpf/features#pkg-variables

Let's call this `PossibleCPU` to match https://pkg.go.dev/runtime#NumCPU?

Reply:
It was super weird I was wondering.